spark/src/test/java/org/apache/iceberg/spark/source/TestSparkDataWrite.java

```diff
@@ -0,0 +1,457 @@
+/*
```

Comment:
Does this file replace `org.apache.iceberg.spark.source.TestParquetWrite` or do u think both are required?

Reply:
yes , I Think this file replace `org.apache.iceberg.spark.source.TestParquetWrite`